ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -1234,6 +1265,107 @@ const triggerSet: TriggerSet<Data> = {
+        if (matches.effectId === 'D72')
+          data.trioDebuff[matches.target] = 'near';
+        if (matches.effectId === 'D73')
+          data.trioDebuff[matches.target] = 'distant';
+      },
+    },
+    {
+      id: 'TOP P5 Delta Debuffs',
+      type: 'Ability',
+      // This is on the Oversampled Wave Cannon ability when there is roughly ~13s left on debuffs.
+      netRegex: { id: '7B6D', capture: false },
+      condition: (data) => data.phase === 'delta',
+      durationSeconds: 8,
+      suppressSeconds: 5,
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          ...nearDistantOutputStrings,
+          unmarkedBlue: {
+            // Probably near baits, but you never know.
+            en: 'Unmarked Blue',
+          },
+        };
+
+        const myDebuff = data.trioDebuff[data.me];
+        if (myDebuff === 'near')
+          return { alertText: output.near!() };
+        if (myDebuff === 'distant')
+          return { alertText: output.distant!() };
+
+        const myColor = data.deltaTethers[data.me];
+        if (myColor === undefined)
+          return;
+
+        // TODO: should we call anything out for greens here??
+        if (myColor === 'blue')
+          return { infoText: output.unmarkedBlue!() };
+      },
+    },
+    {
+      id: 'TOP P5 Sigma Debuffs',
+      type: 'Ability',
+      // This is on the Storage Violation damage, with roughly ~24s on debuffs.
+      netRegex: { id: '7B04', capture: false },
+      condition: (data) => data.phase === 'sigma',
+      durationSeconds: (data) => data.trioDebuff[data.me] === undefined ? 5 : 16,
+      suppressSeconds: 5,
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          ...nearDistantOutputStrings,
+          noDebuff: {
+            en: '(no debuff)',
+          },
+        };
+
+        const myDebuff = data.trioDebuff[data.me];
+        if (myDebuff === 'near')
+          return { alertText: output.near!() };
+        if (myDebuff === 'distant')
+          return { alertText: output.distant!() };
+        return { infoText: output.noDebuff!() };
+      },
+    },
+    {
+      id: 'TOP P5 Omega Debuffs',
+      // First In Line: ~32s duration, ~12s left after 2nd dodge
+      // Second In Line: ~50s duration, ~15s left after final bounce
+      type: 'GainsEffect',
+      netRegex: { effectId: ['D72', 'D73'] },
+      condition: (data, matches) => data.phase === 'omega' && matches.target === data.me,
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) > 40 ? 35 : 20,
+      durationSeconds: 8,
+      alertText: (_data, matches, output) => {
+        if (matches.effectId === 'D72')
+          return output.near!();
+        if (matches.effectId === 'D73')
+          return output.distant!();
+      },
+      outputStrings: nearDistantOutputStrings,
```

Comment:
When I run npx eslint against the file, I get errors on these lines:
 1336:18  error  no 'near' in 'output'     rulesdir/cactbot-output-strings
 1338:18  error  no 'distant' in 'output'  rulesdir/cactbot-output-strings
 
 No errors seen in npm run.

Reply:
Hmm, this is weird.  I do not see any lint errors locally or on the bot.

(What is `npx eslint`? There's no `npm run eslint` rule.  There is `npm lint`?)

Reply:
Sorry, I meant the npm start output does not show errors, but npx eslint on my ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts file with this trigger present produces its

Reply:
What command are you running to see this?

Reply:
from within my cactbot directory:
npx eslint ui\raidboss\data\06-ew\ultimate\the_omega_protocol.ts

Reply:
Hmm, that's weird.  I never use that command, but running that does not show any errors for me.  Is your repository up to date? There were some fixes to the output strings lint rules to improve this.